roles/pulp_database_config/tasks/generate_database_fields_key.yml

```diff
@@ -1,4 +1,19 @@
 ---
+- name: Ensure python-cryptography is installed
+  package:
+    name: '{{ pulp_python_cryptography }}'
+    state: present
+  become: true
+
+- name: Create cert directory to hold token authentication key
+  file:
+    path: "{{ pulp_certs_dir }}"
+    state: directory
+    owner: "{{ pulp_user }}"
+    group: "{{ pulp_group }}"
+    mode: 0700
+  become: true
+
```

Comment:
Import is now broken.

You need to move this into pulp_database_config's main.yml. Before the import task.